app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  
+  def formatted_date(datetime)
```

Comment:
it is formatting_date so it should be `format_date`

Reply:
done